CONTRIBUTION_POLICY.md

```diff
@@ -0,0 +1,66 @@
+
+## Code of Conduct
+
+Before contributing, please familiarize yourself with our [Code of Conduct](CODE_OF_CONDUCT.md), which outlines the expected behavior and responsibilities of all contributors within the CompuCell3D community.
+
+## Contribution Process
+
+1. **Fork the Repository:**
+   - Fork the CompuCell3D repository to your GitHub account.
+
+2. **Create a Branch:**
+   - Create a new branch for your contribution. Use a descriptive name that reflects the purpose of your changes.
+   - Follow git flow model: https://danielkummer.github.io/git-flow-cheatsheet/
+
+3. **Work on Your Contribution:**
+   - Implement your changes, following the coding standards and guidelines outlined in the project.
+   - For explanation on how the code is written, how to compile cc3d and how to create new modules, please see our [Developers' Manual](https://compucell3ddevelopersmanual.readthedocs.io/en/latest/) 
+
+4. **Write Tests:**
+   - If applicable, write tests for your code to ensure its correctness.
+   - IF you are stuck on this issue, just issue your Pull Request and talk to us. We will help you out with this item
+
+5. **Submit a Pull Request (PR):**
+   - Open a pull request from your forked branch to the main CompuCell3D repository. Provide a clear title and description of your changes.
+   - If you are just getting started and woudl like to learn what Pull Request is and how to contribute to open source project please see this [video](https://www.youtube.com/watch?v=8A4TsoXJOs8&ab_channel=WebDevCody)
+
+6. **Code Review:**
+   - Participate in the code review process, addressing any feedback provided by maintainers and community members.
+
+7. **Continuous Integration (CI):**
+   - Ensure that your changes pass the project's continuous integration checks.
+   - For now this is somewhen less important
+
+8. **Merge:**
+   - Once your contribution is approved, it will be merged into the develop branch and subsequently it will become part of the official CompuCell3D release.
+
+## Coding Standards
+
+Follow the coding standards outlined in the [CONTRIBUTING.md](CONTRIBUTING.md) file in the project repository. These standards cover code formatting, naming conventions, and other coding best practices.
+
+## Reporting Issues
+
+If you encounter a bug or have a feature request, please submit an issue on our [GitHub Issues page](https://github.com/CompuCell3D/CompuCell3D/issues). Provide detailed information, including a clear description, steps to reproduce, and any relevant logs or error messages.
+
+## Communication
+
+- **GitHub Discussions:**
+  - Engage in discussions on the [GitHub Discussions page](https://github.com/CompuCell3D/CompuCell3D/issues) for general questions, ideas, and community interaction.
+
+- **Slack Chat:**
```

Comment:
Add a link to join. I guess it would be a new Slack group? I'm in the 2023 Workshop Slack. 
Add the Reddit page: https://www.reddit.com/r/CompuCell3D